django/thunderstore/api/cyberstorm/tests/test_user.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+import pytest
+from django.http import Http404, HttpRequest
+from rest_framework.test import APIClient
+
+from thunderstore.api.cyberstorm.views.user import UserAPIView
+from thunderstore.community.models import CommunitySite
+from thunderstore.repository.models.team import TeamMember
+
+
+@pytest.mark.django_db
+def test_UserAPIView_regular_usage(
```

Comment:
use snake case, also no need to explicitly refer to the name of the view since that could change. Refer to the concept instead (in this case, the cyberstorm API user view, so e.g. `test_api_cyberstorm_user_get` in this case).

The order of concepts in the test names should go from less detail on the left side to more detail on the right side, as it makes for easy test run scoping. E.g. to run all API tests you can use `test_api` prefix filter on test names, and to narrow down to cyberstorm API you'd use `test_api_cyberstorm` and so on.

This applies to all test names in this PR for that matter.